masterlist.yaml

```
@@ -20721,21 +20415,21 @@ plugins:
       - 'Dwemer Certified.esp'
       - 'More Exp Chance.esp'
       - 'Requiem.esp'
-      - 'Requiem - Bugsmasher Edition.esp'
-      - 'Requiem - Legendary Bugsmasher Edition.esp'
       - name: 'Animated Weapon Enchants.esp'
         condition: 'file("EnchantingAwakenedAWEPatch.esp")'
       - name: 'SeeEnchs.esp'
         condition: 'file("EnchantingAwakenedSeeEnchPatch.esp")'
     msg:
-      - <<: *patchIncluded
-        condition: 'active("Requiem.esp") and not active("EnchantingAwakenedRequiem.esp")'
-        subs: [ 'Requiem' ]
+      - <<: *patchUnavailable_Requiem
   - name: 'EnchantingAwakenedRequiem.esp'
     dirty:
       - <<: *dirtyPlugin
         crc: 0xe410b8ec
         itm: 3
+    msg:
+      - <<: *incVersionX
+        condition: 'version("Requiem.esp", "2.0.0", >=)'
+        subs: [ 'Requiem.esp' ]
```

We also order the `inc` key (& the `msg` key for that matter) before the `dirty` key.

```suggestion
    inc:
      - name: 'Requiem.esp'
        display: 'Requiem v2.0.0+'
        condition: 'version("Requiem.esp", "2.0.0", >=)'
    dirty:
      - <<: *dirtyPlugin
        crc: 0xe410b8ec
        itm: 3
```